wire-ios/Wire-iOS/Sources/AppDelegate.swift

```diff
@@ -303,6 +303,8 @@ private extension AppDelegate {
             let appVersion = Bundle.main.infoDictionary?[kCFBundleVersionKey as String] as? String,
             let url = Bundle.main.url(forResource: "session_manager", withExtension: "json"),
             let configuration = SessionManagerConfiguration.load(from: url),
+            let countlyKey = Bundle.countlyAppKey,
+            let host = BackendEnvironment.shared.countlyURL,
```

Comment:
This means that if there must be a countly key and host configure otherwise the app wouldn't run. But not everyone will use countly. You'll need to pass these in as optionals.